README.md

```diff
@@ -29,18 +29,29 @@
 
 ### Local Machine 
 
-- Set the application deployment (assuming a live K8s cluster w/ app) UUID as ENV (APP_UUID)
-
-- Set the ChaosEngine CR name as ENV (CHAOSENGINE) 
-  - For CR spec, see: https://github.com/litmuschaos/chaos-operator/blob/master/deploy/crds/chaosengine.yaml
-
-- If the experiments are not executed, apply the ChaosResult CRs manually 
-  - For CR spec, see: https://github.com/litmuschaos/chaos-operator/blob/master/deploy/crds/chaosresult.yaml
-
-- Run the exporter container (litmuschaos/chaos-exporter:ci) on host network. It is necessary to mount the kubeconfig
-  & override entrypoint w/ `./exporter -kubeconfig <path>`
-
-- Execute `curl 127.0.0.1:8080/metrics` to view metrics
+- Fork this repo!
+- Edit the GOPATH variable and help it point to your workspace.
+- Clone the repository where you set the workspace.
+- Use the `git clobe repo-name` ,or just copy the git clone URL avaliable on the right side.
+- Set an APP_UUID in the `~/.bashrc` or the `~/.profile`, add this command to set a default
+- Now, start your Local Cluster, (this guide helps in `minikube` but can be used for other offline clusters as well)
+- Create Kubernetes CR's(Custom Resources) for the litmus operator, link down below:
+  - For ChaosEngine : https://github.com/litmuschaos/chaos-operator/blob/master/deploy/crds/chaosengine_crd.yaml
+  - For ChaosResult : https://github.com/litmuschaos/chaos-operator/blob/master/deploy/crds/chaosresults_crd.yaml
+  - For ChaosExperiment: https://github.com/litmuschaos/chaos-operator/blob/master/deploy/crds/chaosexperiment_crd.yaml
+- Now, as you have created the CustomResourceDefinition, Now it time to create the CustomResources for these definition above:
```

Comment:
Lets set the expectation that these steps are provided to mock experiment runs & visualize how actual metrics would be generated. Generally, either exporters work out of the box (ex: node-exporters) or they are very tied to an app (say, mysql). In our case they are tied to the chaos-resources. So, these can be placed in a pre-requisites section, saying: 

- The following steps are required to create sample chaos-related custom resources in order to visualize the metrics gathered by the chaos exporter

Reply:
Might make sense to provide link to the operator repo and/or docs for the user to figure out what indeed are these chaosengine/experiment/results..

Reply:
The need/reason for setting APP_UUID has to be explained, saying - APP_UUID is derived from the app to be added as a metric label.